conjure-runtime/src/builder.rs

```diff
@@ -244,12 +319,51 @@ impl Builder {
         self
     }
 
+    /// Returns the builder's configured host metrics registry.
+    pub fn get_host_metrics(&self) -> Option<&Arc<HostMetricsRegistry>> {
+        self.host_metrics.as_ref()
+    }
+
+    /// Sets the raw client builder.
+    ///
+    /// Defaults to `DefaultRawClientBuilder`.
+    pub fn with_raw_client_builder<U>(self, raw_client_builder: U) -> Builder<U> {
```

Comment:
This has to take self by value since it's changing the type parameter, which doesn't match up with the rest of the builder methods taking self by reference. However, self by reference is generally preferable and people really shouldn't need to use this method very often anyway.

Reply:
This makes sense. Thanks for the explanation.